ui/components/app/signature-request/signature-request.component.js

```diff
@@ -136,7 +153,7 @@ export default class SignatureRequest extends PureComponent {
           </div>
         ) : null}
         <Message
-          data={sanitizeMessage(message, primaryType, types)}
+          data={{ ...sanitizeMessage(message, primaryType, types), domain }}
```

Comment:
This seems dangerous. `domain` is a valid field for the message itself, and this would override that field (if it existed).

This also would display the domain separator data as part of the message, which it is not.

When I suggested displaying the domain information as part of the `Message`, I meant that it could be passed as a separate prop and displayed separately (_somehow_ - I don't know how we should display it).

Reply:
domain is not displayed in old component even

Reply:
I had brought it up here because previously the `domain.name` was shown where the site icon is now shown.

Overall this is a step forward because the way we were displaying `domain.name` was misleading, but there is a sense in which this is a regression because now we're not showing it anywhere. So if a user was looking to verify the `domain.name` of the message, that would be impossible now.

But yes, maybe you are right, we can deal with it in a separate PR. Hiding it seems less bad than displaying it in a misleading way.